modules/olm-updating-operator-catalog-image.adoc

```diff
@@ -0,0 +1,151 @@
+image is already configured for use with OperatorHub.
+
+.Prerequisites
+
+* A Linux workstation with unrestricted network access
+ifeval::["{context}" == "olm-restricted-networks"]
+footnoteref:[BZ1771329]
+endif::[]
+* `oc` version 4.3+
+* `podman` version 1.5.1+
+* Access to mirror registry that supports
+link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2], which
+must be exposed with a certificate that is trusted by the workstation
+* OperatorHub configured to use custom catalog images
+
+.Procedure
+
+. On the workstation with unrestricted network access, authenticate with the
+target image registry:
++
+----
+$ podman login <registry_host_name>
+----
+
+. Build a new catalog image based on the `redhat-operators` catalog from
+link:https://quay.io/[quay.io], tagging and pushing it to your mirror registry:
++
+----
+$ oc adm catalog build \
+    --appregistry-endpoint https://quay.io/cnr \
+    --appregistry-org redhat-operators \ <1>
+    --to=<registry_host_name>:<port>/<namespace>/redhat-operators:v2 <2>
+
+INFO[0013] loading Bundles                               dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605
+INFO[0013] directory                                     dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605 file=manifests-829192605 load=bundles
+INFO[0013] directory                                     dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605 file=3scale-operator load=bundles
+INFO[0013] found csv, loading bundle                     dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605 file=3scale-operator.v0.3.0.clusterserviceversion.yaml load=bundles
+INFO[0013] loading bundle file                           dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605/3scale-operator file=3scale-operator.package.yaml load=bundle
+INFO[0013] loading bundle file                           dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605/3scale-operator file=3scale-operator.v0.3.0.clusterserviceversion.yaml load=bundle
+...
+Uploading ... 244.9kB/s
+Pushed sha256:f73d42950021f9240389f99ddc5b0c7f1b533c054ba344654ff1edaf6bf827e3 to example_registry:5000/openshift/redhat-operators:v2
+----
+<1> Organization (namespace) to pull from an App Registry instance.
+<2> Image repository tag to apply to the built catalog image. Specify a namespace and give it a tag, for example, `v2` because it is the updated catalog.
+
+. Mirror the contents of your catalog to your target registry. The following
+`oc adm catalog mirror` command extracts the contents of your custom Operator
+catalog image to generate the manifests required for mirroring and mirrors the
+images to your registry:
++
+----
+$ oc adm catalog mirror \
+    <registry_host_name>:<port>/<namespace>/redhat-operators:v2 \ <1>
+    <registry_host_name>:<port>/openshift <2>
```

Comment:
has anyone tested this step?

I suspect this will generate manifest names that not all target registries support, because it includes the `openshift` namespace.

Reply:
What's a better example namespace to show here? @shawn-hurley

Reply:
I think you need to remove `/openshift` 

so `<registry_host_name>:<port>` should be all you need.